src/pages/commits/InactiveCommits/index.tsx

```diff
@@ -120,25 +122,38 @@ const ButtonContainer = styled.div`
   justify-content: center;
 `;
 
-const InactiveCommitLine = styled.div`
-  height: ${commitChartHeight}px;
-  border: 1px dashed ${gray.light1};
-`;
-
 const ButtonText = styled(Disclaimer)`
   margin-top: ${size.xs};
   text-align: center;
   color: ${gray.dark2};
   font-weight: bold;
 `;
 
-const TitleText = styled(Body)`
-  margin-bottom: ${size.s};
+const HiddenCommitsWrapper = styled.div`
+  align-self: center;
+  padding: ${size.xs} 0;
+  opacity: 0.5;
 `;
 
-const Container = styled.div`
-  display: flex;
-  justify-content: center;
+const TooltipTitleText = styled.div`
+  margin-bottom: ${size.xs};
+  font-weight: bold;
 `;
 
-const MAX_COMMIT_COUNT = 5;
+const CommitText = styled.div`
+  padding: ${size.xxs} 0;
+  word-break: break-all;
+  font-size: 13px;
```

Comment:
Does `fontSize.m` work here? 14px?

Reply:
I think I'll keep it because `14px` looks a little ugly 😔